assets/js/state/sagas/index.js

```diff
@@ -95,14 +92,6 @@ function* initialDataFetch() {
 
   yield fork(checkApiKeyExpiration);
 
-  const {
-    data: { eula_accepted, premium_subscription },
-  } = yield call(get, '/settings');
-
-  if (!eula_accepted) {
-    yield put(setEulaVisible());
-  }
-
   if (premium_subscription) {
```

Comment:
`premium_subscription` is never available. I guess also this condition can go with also `yield put(setIsPremium());` and the related piece of state/saga/selector if any.